git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH git-gud 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[option1] [option2]
+.SH DESCRIPTION
+.B git-gud
+teaches you how to use Git with fun interactive tutorials.
+Use the following command to create a Git repository and start the first level:
+.B python\ \-m\ gitgud\ start
```

Comment:
We don't really use this (we can, but we don't advertise it). Git gud is actually registered as a program. If you look in wherever you have python installed, you'll see a `git-gud` executable which is run when you type in `git gud`

Also, the start command is out of date. It's `git gud init`.

Reply:
Okay, I will make the change from `python\ \-m\ gitgud\ start` to `git gud init`, possibly moving the added change to a more relevant section.

Reply:
Fixed